Cargo.toml

```diff
@@ -21,3 +21,6 @@ time = "*"
 unicase = "*"
 unsafe-any = "*"
 url = "*"
+
+[profile.bench]
+opt-level = 0
```

Comment:
Oh dangit. I ahead set that when I was fixing the bench marks, to compile faster.